Source/Internal/RequestController/HARequestController.swift

```diff
@@ -5,14 +5,17 @@ internal struct HARequestControllerAllowedSendKind: OptionSet {
 
     static let webSocket: Self = .init(rawValue: 0b1)
     static let rest: Self = .init(rawValue: 0b10)
-    static let all: Self = [.webSocket, .rest]
+    static let sttData: Self = .init(rawValue: 0b10)
```

Comment:
`0b100` - `0b10` is already taken above

Reply:
0b11 then right? Why aren't we just using integers here?